### PR TITLE
Adding tables to readme

### PR DESCRIPTION
**Pull request by Shimwell**
Just adding a little table to the readme as a guide that shows what we have covered and the different options for downloading

Also I noticed the https://openmc.mcs.anl.gov/ website as two scripts for JEFF 3.3, just wondering if there is a difference

probably best to go here to see the rendered table https://github.com/openmc-dev/data/tree/adding_tables_to_readme

### CODE REVIEW

**Review by paulromano — CHANGES_REQUESTED** [review]
This is a little misleading because the "MCNP" libraries are actually ENDF/B. It's just that we have multiple ENDF/B libraries (one processed by LANL, one processed by NNDC, one processed by us). Maybe have another column to indicate what organization did the processing?

Also you can use the much nicer openmc.org instead of openmc.mcs.anl.gov.

File: README.md [hunk 1]
```diff
@@ -6,3 +6,18 @@ ACE libraries (such as those produced by LANL) whereas generate scripts use NJOY
 process ENDF files directly. Note that unless you are interested in making a
 customized library, you can find pregenerated HDF5 libraries at
 https://openmc.mcs.anl.gov.
+
+
+| Library | Release | Download from [openmc.mcs.anl.gov](https://openmc.mcs.anl.gov/) | Download ACE files and convert HDF5 | Download ENDF files and generate HDF5 | Convert local ACE files |
```
Comment (Shimwell) [inline README.md:11]:
```suggestion
| Library | Release | Download from [openmc.org](https://openmc.org/) | Download ACE files and convert HDF5 | Download ENDF files and generate HDF5 | Convert local ACE files |
```

**Comment by Shimwell** [comment]
Had a go at adding the processed by column. I think I have confused myself as well in the process. But perhaps it is a little closer

**Comment by Shimwell** [comment]
Also I noticed there is reference to a openmc-get-multipole-data in the openmc docs https://docs.openmc.org/en/latest/examples/nuclear-data.html#Windowed-multipole

This script does not feature in the table and is not in the data repository but perhaps it should? 

Perhaps good to mention this repo somewhere in the data repository README https://github.com/mit-crpg/WMP_Library

File: README.md [hunk 2]
```diff
@@ -6,3 +6,18 @@ ACE libraries (such as those produced by LANL) whereas generate scripts use NJOY
...
+
+
+| Library | Release | Processed by | Download from [openmc.org](https://openmc.org/) | Download ACE files and convert HDF5 | Download ENDF files and generate HDF5 | Convert local ACE files |
+|-|-|-|-|-|-|-|
+| CENDL | 3.1 |  |  |  | generate_cendl.py |  |
+| ENDF | B-VII.0 | LANL | :heavy_check_mark: |  |  | convert_mcnp70.py |
+| ENDF | B-VII.1 | LANL | :heavy_check_mark: |  |  | convert_mcnp71.py |
+| ENDF | B-VII.1 | NNDC | :heavy_check_mark: | convert_nndc71.py | generate_endf71.py |  |
+| ENDF | B-VIII.0 | LANL | :heavy_check_mark: |  |  | convert_lib80x.py |
+| ENDF | B-VIII.0 | NNDC | :heavy_check_mark: |  | generate_endf80.py |  |
```
Comment (paulromano) [inline README.md:18]:
```suggestion
| ENDF/B | VII.0 | LANL | :heavy_check_mark: |  |  | convert_mcnp70.py |
| ENDF/B | VII.1 | LANL | :heavy_check_mark: |  |  | convert_mcnp71.py |
| ENDF/B | VII.1 | NNDC | :heavy_check_mark: | convert_nndc71.py | generate_endf71.py |  |
| ENDF/B | VIII.0 | LANL | :heavy_check_mark: |  |  | convert_lib80x.py |
| ENDF/B | VIII.0 | NNDC | :heavy_check_mark: |  | generate_endf80.py |  |
```

**Comment by paulromano** [comment]
`openmc-get-multipole-data` hasn't existed for quite some time. It was removed after the library itself was made available at the github repo you've mentioned. Probably a good idea to mention it on the readme!

File: README.md (hunk 2 above)
Comment (Shimwell) [reply README.md]: Thanks

File: README.md [hunk 3]
```diff
@@ -2,7 +2,24 @@
...
 This repository contains a collection of scripts for generating HDF5 data
 libraries that can be used with OpenMC. Some of these scripts convert existing
-ACE libraries (such as those produced by LANL) whereas generate scripts use NJOY to
-process ENDF files directly. Note that unless you are interested in making a
-customized library, you can find pregenerated HDF5 libraries at
-https://openmc.mcs.anl.gov.
+ACE libraries (such as those produced by LANL) whereas generate scripts use
+NJOY to process ENDF files directly. Note that unless you are interested in
+making a customized library, you can find pregenerated HDF5 libraries at
+https://openmc.mcs.anl.gov. Another source of data libraries for OpenMC is the
```
Comment (paulromano) [inline README.md:8]:
```suggestion
https://openmc.org. Another source of data libraries for OpenMC is the
```

File: README.md [hunk 4]
```diff
@@ -2,7 +2,24 @@
...
+[Windowed Multipole Library](https://github.com/mit-crpg/WMP_Library)
+repository which allows on-the-fly Doppler broadening to an arbitrary
```
Comment (paulromano) [inline README.md:10]:
```suggestion
repository which enables on-the-fly Doppler broadening to an arbitrary
```

**Review by paulromano — CHANGES_REQUESTED** [review]
Two more nitpicks here and I think we're done :smile: Thanks @Shimwell!

**Review by paulromano — APPROVED** [review]

**Pull request merged**